Traceback (most recent call last):
  File "file-test-data/extras/testTypesHigherOrderFuns3.py", line 38, in <module>
    homePoints: Callable[[GameResult], int] = mkGamePoints(42)

WyppTypeError: 42

Der Aufruf der Funktion `mkGamePoints` erwartet einen Wert vom Typ `Callable[[int, int], bool]` als erstes Argument.
Aber der Ã¼bergebene Wert hat den Typ `int`.

## Datei file-test-data/extras/testTypesHigherOrderFuns3.py
## Fehlerhafter Aufruf in Zeile 38:

homePoints: Callable[[GameResult], int] = mkGamePoints([0;31m[1m42[0;0m)

## Typ deklariert in Zeile 35:

def mkGamePoints([0;31m[1mcmp: Callable[[int, int], bool][0;0m) -> Callable[[GameResult], int]: